static/js/sefaria/strings.js

```diff
@@ -114,6 +114,8 @@ const Strings = {
     // Topic Images
     "Illustrative image" : "תמונה להמחשה",
 
+    // Topic Landing Page
+    "Featured Topic": "Featured Topic",
```

Comment:
For now Hebrew can be `הנושא היומי` I guess

Reply:
I'm waiting to get an "official" translation from Mickey